docs/ql-documentation/learn-ql/cpp/introduce-libraries-cpp.rst

```diff
@@ -4,114 +4,686 @@ Introducing the C/C++ libraries
+| ``int`` **scanf** ``(const char *format, ...);``                                                                                                                                                                                                                                                                                                                                                                                            | `Scanf <https://help.semmle.com/qldoc/cpp/semmle/code/cpp/commons/Scanf.qll/type.Scanf$Scanf.html>`__                                                                 |                                                                                                                                                                                                                                                                                   |
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``int`` **snscanf** ``(const char *source, size_t length, const char *format, ...)``                                                                                                                                                                                                                                                                                                                                                        | `Snscanf <https://help.semmle.com/qldoc/cpp/semmle/code/cpp/commons/Scanf.qll/type.Scanf$Snscanf.html>`__                                                             |                                                                                                                                                                                                                                                                                   |
++-------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
`#undef`

Reply:
(github seems to be unsure whether this comment should be on line 99 or 672; it should be on the latter)

Reply:
I'm not sure I follow.  Are you getting some sort of error or visual artifact when rendering the document?  The Sphinx compiler is not producing any errors, and the document displays fine (for me) both on GitHub and directly in the browser.

Reply:
Now I see the `#under`... :-)